graylog2-web-interface/src/preflight/hooks/useConfigurationStep.test.tsx

```diff
@@ -136,6 +154,14 @@ describe('useConfigurationStep', () => {
       }],
     });
 
+    asMock(useRenewalPolicy).mockReturnValue({
+      ...useRenewalPolicyResult,
+      data: {
+        mode: 'AUTOMATIC',
+        certificate_lifetime: 'P30D',
+      },
+    });
+
     const { result, waitFor } = renderHook(() => useConfigurationStep());
 
     await waitFor(() => expect(result.current).toEqual({
```

Comment:
Have you considered adding a test for the renewal policy step?

Reply:
:white_check_mark: